modules/ROOT/pages/outbound-message-routing.adoc

```diff
@@ -14,7 +14,14 @@ When messages are received by the endpoint, Partner Manager routes the payload t
 ** Relative path for HTTP(s) endpoints
 ** File name for SFTP
 * The payload is validated against the schema supplied during the message type definition.
-* Partner Manager then leverages the DataWeave map supplied during the source at host message type definition to extract the partner reference ID, host reference ID, and business key.
+* Partner Manager leverages the DataWeave map supplied during the source at host message type definition to extract the partner reference ID, host reference ID, and business key.
```

Comment:
Can we reword this...noun?
"the DataWeave map supplied during the source at host message type definition"
Also, "leverages to extract" is wordy and vague.
Why not something simple like "Partner Manager uses the supplied DataWeave map to obtain the partner and host reference IDs and the business key."

Reply:
Done